### ISSUE TITLE
Incorrect value `rule.raws.semicolon`

### ISSUE THREAD

**Issue by evilebottnawi** [shellscape/postcss-less#70]
* Node Version: `7.10.0`
* NPM Version: `4.2.0`
* postcss-less Version: `0.16.1`

If you have a large amount of code to share which demonstrates the problem you're experiencing, please provide a link to your
repository rather than pasting code. Otherwise, please paste relevant short snippets below.

### LESS

```less
a { .mixin; .mixin2; }
```

### JavaScript

```js
import postCssLess from 'postcss-less';
const less = `a { .mixin; .mixin2; }`;
const root = postCssLess.parse(less);
```

### Expected Behavior

`root.nodes[0].raws.semicolon` should be `true`.
`root.nodes[0].raws.after` should be ` `.

### Actual Behavior

`root.nodes[0].raws.semicolon` is `false`.
`root.nodes[0].raws.after` is `;  `.

### How can we reproduce the behavior?

Use example in `javascript` section.

Issue was created based on https://github.com/stylelint/stylelint/pull/2547.
Information about https://github.com/stylelint/stylelint/pull/2547#issuecomment-299712329.
Information about expected behavior based on `postcss` docs http://api.postcss.org/Node.html#raws

**Comment by shellscape** [comment]
Thanks for creating the issue :)

**Comment by shellscape** [comment]
Well I've managed to correct the `semicolon` property, but I can't seem to figure out where the heck the code is prepending `;` to `after`

**Comment by shellscape** [comment]
@evilebottnawi I spent about 4 hours on this tonight, and while I've got the raws properties correct according to the spec, the stringifying is completely hosed. I'm afraid this one is going to have to wait for someone better than I to resolve.

**Comment by evilebottnawi** [comment]
@shellscape your can ask for helping @ai or someone from `postcss` (I hope they find time to help)

**Comment by ai** [comment]
Hi. Yeap. Ask you direct question.

**Comment by shellscape** [comment]
@ai I'm sorry, I don't understand. Could you rephrase?

**Comment by ai** [comment]
I mean ask your question, I will help you.

**Comment by shellscape** [comment]
I didn't have a question really. I was telling @evilebottnawi that the solution to the problem is a bit beyond me and that I wouldn't be able to fix it, that someone else would have to step in and submit a fix. But thank you for offering.
